helm/templates/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: indexd-conf
+    labels:
+        app.kubernetes.io/name: {{ include "k8s-service.name" . }}
+        helm.sh/chart: {{ include "k8s-service.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+{{ (tpl (.Files.Glob "files/indexd.conf").AsConfig .) | indent 4 }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: indexd-local-settings
+    labels:
+        app.kubernetes.io/name: {{ include "k8s-service.name" . }}
+        helm.sh/chart: {{ include "k8s-service.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+{{ (.Files.Glob "files/local_settings.py").AsConfig | indent 4 }}
```

Comment:
Is this file supposed to exist under the `helm/files` directory? Cos this is where we can add code to conditionally check if the proper user exists and then create it, so indexd is guaranteed to always have a user

Reply:
Ah man, that file was being ignored by .gitignore. I have forced it to be included now. We can probably extend it in another PR to do all the user stuff, and keep it vanilla for now.

Reply:
its like one or two line [see here](https://github.com/NCI-GDC/gib/blob/09a83abbe3e5778f6ceb4194d540c91faa7b8d19/.docker/indexd/settings.py#L37-L40) but yeah we can do that later